daemon/cluster/cluster.go

```diff
@@ -1570,8 +1529,7 @@ func (c *Cluster) CreateNetwork(s apitypes.NetworkCreateRequest) (string, error)
 	return r.Network.ID, nil
 }
 
-// RemoveNetwork removes a cluster network.
-func (c *Cluster) RemoveNetwork(input string) error {
+func (c *Cluster) updateClusterObject(updateFn func(ctx context.Context, state nodeState) error) error {
```

Comment:
The github diff doesn't really show it properly, but `updateClusterObject` does contain this piece of code 😉